app/views/admin/payroll_run_downloads/show.html.erb

```diff
@@ -13,7 +13,7 @@
       <p class="govuk-body">
         This month's payroll file was downloaded on
         <%= l(@payroll_run.downloaded_at) %> by
-        <%= @payroll_run.downloaded_by %>.
+        <%= user_details(@payroll_run.downloaded_by, include_line_break: false) %>.
```

Comment:
The option which is used here isn't added until the [next commit](https://github.com/DFE-Digital/dfe-teachers-payment-service/pull/747/commits/b6c9c562bb9f4743a3dad61d00a6702722e526e0) - are these out of order?